src/main/scala/interrupts/CrossingHelper.scala

```diff
@@ -27,6 +30,8 @@ case class IntOutwardCrossingHelper(name: String, scope: LazyScope, node: IntOut
         IntSyncRationalCrossingSink() :*=* IntSyncNameNode(name) :*=* scope { IntSyncNameNode(name) :*=* IntSyncCrossingSource(alreadyRegistered) } :*=* node
       case SynchronousCrossing(buffer) =>
         IntSyncSyncCrossingSink() :*=* IntSyncNameNode(name) :*=* scope { IntSyncNameNode(name) :*=* IntSyncCrossingSource(alreadyRegistered) } :*=* node
+      case CreditedCrossing(CreditedDelay(sourceDebit, _), CreditedDelay(sinkDebit, _)) =>
+        IntSyncSyncCrossingSink(/*sinkDebit==0*/) :*=* IntSyncNameNode(name) :*=* scope { IntSyncNameNode(name) :*=* IntSyncCrossingSource(sourceDebit==0) } :*=* node
```

Comment:
why is sinkDebit ignored here?

Reply:
The IntSyncCrossingSink does not support a register stage. I could change that. Should I?

Reply:
No I guess I would just leave it, though I confess I don't really understand the semantics of IntSync are as it relates to what circuits present "safe" crossings. Would a user explicitly controlling `sinkDebit` be surprised at the overall result in either direction?